server-nodejs/extensions/jwt.js

```diff
@@ -132,5 +132,36 @@ module.exports = function (app) {
         res.status(200).json(req.user);
     });
 
+
+    app.route('/api/createToken').post(function (req, res, next) {
```

Comment:
No because GET requests are used to retrieve data, they can be cached by browsers, and here we don’t want the token to be cached + potentially modifies the last login time